docs/config.rst

```diff
@@ -46,4 +51,17 @@ Any configuration key/value listed here may also be used in the specific plugin
 gcp.gdns
 ~~~~~~~~
 
-All configuration options above may be used here. There are no specific DNS-related configuration options.
+All configuration options above in the general ``[gcp]`` may be used here. There are no specific DNS-related configuration options.
+
+gcp.gpubsub
+~~~~~~~~~~~
+
+All configuration options above in the general ``[gcp]`` may be used here. Additional Google Cloud Pub/Sub-related configuration is needed:
+
+.. option:: topic="STR"
+
+    `Required`: Google Pub/Sub topic to receive the publish change messages.
+
+.. attention::
```

Comment:
I elected to have this `attention` twice since folks (myself included) are often not great at reading docs in detail 🙃